"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on a [[""user-provided value""|""relative:///examples/example_plugin/example_plugin/api/views.py:44:49:44:60""]].","/examples/example_plugin/example_plugin/api/views.py","44","13","44","104"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/ipam/api/fields.py:16:43:16:43""]] flows to this location and may be exposed to an external user.","/nautobot/ipam/api/fields.py","17","47","17","47"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/core/api/mixins.py:94:30:94:30""]] flows to this location and may be exposed to an external user.","/nautobot/core/api/mixins.py","95","35","95","35"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/core/api/parsers.py:60:29:60:31""]] flows to this location and may be exposed to an external user.","/nautobot/core/api/parsers.py","61","30","61","37"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/core/api/serializers.py:800:34:800:34""]] flows to this location and may be exposed to an external user.","/nautobot/core/api/serializers.py","801","39","801","39"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/core/api/serializers.py:937:45:937:49""]] flows to this location and may be exposed to an external user.","/nautobot/core/api/serializers.py","938","39","938","48"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/core/api/serializers.py:957:45:957:49""]] flows to this location and may be exposed to an external user.","/nautobot/core/api/serializers.py","958","39","958","48"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:560:37:560:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:560:37:560:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:560:37:560:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:560:37:560:37""]] flows to this location and may be exposed to an external user.","/nautobot/extras/api/serializers.py","561","55","561","68"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:602:37:602:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:602:37:602:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:602:37:602:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///nautobot/extras/api/serializers.py:602:37:602:37""]] flows to this location and may be exposed to an external user.","/nautobot/extras/api/serializers.py","603","55","603","78"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/core/api/utils.py:196:32:196:45""]] flows to this location and may be exposed to an external user.","/nautobot/core/api/utils.py","203","25","203","28"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/extras/api/views.py:1016:31:1016:31""]] flows to this location and may be exposed to an external user.","/nautobot/extras/api/views.py","1019","25","1019","32"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///nautobot/dcim/api/views.py:509:29:509:29""]] flows to this location and may be exposed to an external user.","/nautobot/dcim/api/views.py","510","38","510","85"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:405:20:405:26""]].","/nautobot/core/views/generic.py","431","41","431","43"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:405:20:405:26""]].","/nautobot/core/views/generic.py","433","37","433","59"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:405:20:405:26""]].","/nautobot/core/views/generic.py","439","37","439","69"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:500:20:500:26""]].","/nautobot/core/views/generic.py","523","33","523","65"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:581:20:581:26""]].","/nautobot/core/views/generic.py","624","41","624","52"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:581:20:581:26""]].","/nautobot/core/views/generic.py","625","37","625","64"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:681:20:681:26""]].","/nautobot/core/views/generic.py","759","37","759","59"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:681:20:681:26""]].","/nautobot/core/views/generic.py","765","37","765","69"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:937:19:937:25""]].","/nautobot/core/views/generic.py","938","25","938","52"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:945:20:945:26""]].","/nautobot/core/views/generic.py","1038","37","1038","64"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:945:20:945:26""]].","/nautobot/core/views/generic.py","1070","29","1070","56"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1105:20:1105:26""]].","/nautobot/core/views/generic.py","1113","29","1113","56"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1105:20:1105:26""]].","/nautobot/core/views/generic.py","1147","45","1147","72"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1205:19:1205:25""]].","/nautobot/core/views/generic.py","1206","25","1206","52"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1208:20:1208:26""]].","/nautobot/core/views/generic.py","1238","37","1238","64"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1208:20:1208:26""]].","/nautobot/core/views/generic.py","1243","33","1243","60"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1208:20:1208:26""]].","/nautobot/core/views/generic.py","1263","29","1263","56"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1328:20:1328:26""]].","/nautobot/core/views/generic.py","1379","41","1379","63"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1328:20:1328:26""]].","/nautobot/core/views/generic.py","1381","41","1381","68"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1417:20:1417:26""]].","/nautobot/core/views/generic.py","1435","29","1435","56"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/generic.py:1417:20:1417:26""]].","/nautobot/core/views/generic.py","1497","37","1497","64"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/mixins.py:370:19:370:30""]].","/nautobot/core/views/mixins.py","413","37","413","64"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/core/views/mixins.py:424:19:424:30""]].","/nautobot/core/views/mixins.py","435","33","435","60"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/ipam/views.py:878:20:878:26""]].","/nautobot/ipam/views.py","884","29","884","57"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/ipam/views.py:935:19:935:25""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/ipam/views.py:938:20:938:26""]].","/nautobot/ipam/views.py","933","29","933","56"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/ipam/views.py:938:20:938:26""]].","/nautobot/ipam/views.py","1025","41","1025","68"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/users/views.py:41:19:41:25""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/users/views.py:56:20:56:26""]].","/nautobot/users/views.py","91","37","91","47"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/users/views.py:260:20:260:26""]].","/nautobot/users/views.py","277","33","277","44"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/dcim/views.py:102:20:102:26""]].","/nautobot/dcim/views.py","123","33","123","42"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/dcim/views.py:2624:20:2624:26""]].","/nautobot/dcim/views.py","2641","37","2641","59"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/dcim/views.py:2624:20:2624:26""]].","/nautobot/dcim/views.py","2643","33","2643","68"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/dcim/views.py:2680:20:2680:26""]].","/nautobot/dcim/views.py","2702","29","2702","64"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:387:20:387:26""]].","/nautobot/extras/views.py","425","41","425","43"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:387:20:387:26""]].","/nautobot/extras/views.py","427","37","427","59"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:387:20:387:26""]].","/nautobot/extras/views.py","433","37","433","69"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:617:20:617:26""]].","/nautobot/extras/views.py","663","41","663","43"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:617:20:617:26""]].","/nautobot/extras/views.py","665","37","665","59"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:617:20:617:26""]].","/nautobot/extras/views.py","671","37","671","69"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:1553:20:1553:26""]].","/nautobot/extras/views.py","1565","25","1565","50"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:1985:20:1985:26""]].","/nautobot/extras/views.py","2021","41","2021","43"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:1985:20:1985:26""]].","/nautobot/extras/views.py","2023","37","2023","59"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///nautobot/extras/views.py:1985:20:1985:26""]].","/nautobot/extras/views.py","2029","37","2029","69"
